sdk/Lusid.Sdk.Tests/tutorials/Ibor/GenericReconciliation.cs

```diff
@@ -0,0 +1,773 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Client;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Ibor
+{
+    [TestFixture]
+    public class GenericReconciliation : TutorialBase
+    {
+        private InstrumentLoader _instrumentLoader;
+        private IList<string> _instrumentIds;
+        private readonly string _portfolioOneScope = "testPortfolio1" + new Guid();
+        private readonly string _portfolioTwoScope = "testPortfolio2" + new Guid();
+        private readonly string _portfolioCode = Guid.NewGuid().ToString();
+        private DateTimeOffset _effectiveAt;
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            _instrumentLoader = new InstrumentLoader(_apiFactory);
+            _instrumentIds = _instrumentLoader.LoadInstruments();
+        }
+
+        /// <summary>
+        /// Perform a reconciliation on two identical portfolios except for the Address Key for trader name being scope dependent.
+        /// I.e., "Transaction/testPortfolio1/TraderName" = "John Doe" and "Transaction/testPortfolio2/TraderName" = "John Doe"
+        /// Unless a mapping rule is provided between these two address keys in the reconciliation then both keys will appear
+        /// as failures in the aggregation.
```

Comment:
I'd suggest as "failures to match" to clarify: It's not 'failed' as such, you have done the comparison, just that the result of the comparison is a failure. We use 'failure' very much to mean the request itself failed and that's not the case.

Reply:
Have changed